Description: Build the test pgms with the package
 This is for autopkgtest.

--- cryfs-0.9.7.orig/CMakeLists.txt
+++ cryfs-0.9.7/CMakeLists.txt
@@ -7,11 +7,6 @@ include(utils.cmake)
 require_gcc_version(4.8)
 require_clang_version(3.7)
 
-# Default value is not to build test cases
-if(NOT BUILD_TESTING)
-    set(BUILD_TESTING OFF CACHE BOOL "BUILD_TESTING")
-endif(NOT BUILD_TESTING)
-
 # Default vaule is to build in release mode
 if(NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE Release CACHE INTERNAL "CMAKE_BUILD_TYPE")
